spec/features/import_booking_bug_data_spec.rb

```diff
@@ -70,24 +72,24 @@
   end
 
   def given_old_booking_bug_data_exists # rubocop:disable Metrics/MethodLength
-    jun_2016 = create(:year_month, date: Time.zone.parse('2016-06-15'))
-    may_2016 = create(:year_month, date: Time.zone.parse('2016-05-15'))
+    jun2016 = create(:year_month, date: Time.zone.parse('2016-06-15'))
```

Comment:
Was this a rubocop correction? This one's a bit weird, can't see anything wrong with the original.